src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,285 @@
+Feature: Descriptors are a set of mechanisms to support flexible enumerations or code sets
+
+        Background:
+            Given the system has this "educationOrganizationCategoryDescriptors"
+                  | codeValue | description | namespace                                               | shortDescription |
+                  | School    | School      | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School           |
+              And the system has this "disabilityDescriptors"
+                  | codeValue       | description          | namespace                            | shortDescription     |
+                  | Disability Test | User Disability Test | uri://ed-fi.org/DisabilityDescriptor | User Disability Test |
+
+        @ignore
+        Scenario: 01 / Ensure clients can create a Descriptor with the required attributes
+             When a POST reques is made to "/ed-fi/disabilityDescriptors" with
+                  """
+                  {
+                      "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                      "codeValue": "Deaf-Blindness",
+                      "shortDescription": "Deaf-Blindness",
+                      "description": "Deaf-Blindness"
+                  }
+                  """
+             Then it should respond with 201
+
+
+        @ignore
+        Scenario: 02 / Ensure clients can retrieve information of a Descriptor
+             When a GET request is made to "/ed-fi/disabilityDescriptors"
+             Then it should respond with 200
+              And the response body is
+                  """
+                  [
+                      {
+                          "id": "{id}"
+                          "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                          "codeValue": "Deaf-Blindness",
+                          "shortDescription": "Deaf-Blindness",
+                          "description": "Deaf-Blindness",
+
+                      },
+                      {
+                          "id": "{id}"
+                          "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                          "codeValue": "Disability Test",
+                          "shortDescription": "User Disability Test",
+                          "description": "User Disability Test",
+                      }
+                  ]
+                  """
+
+        @ignore
+        Scenario: 03 / Ensure clients can update the values of a Descriptor
+            Given the system has this codeValue "Deaf-Blindness"
```

Comment:
This will force the C# developer to write a new step definition to look up that codevalue. 

It will be easier if you simply re-create the item here in the Given, then on the next line we will automatically (in C#) know which `id` value to use. See the modifications I just made in `UpdateResourcesValidation` for example.

Reply:
Recommendation followed